docs/output_files.md

```diff
@@ -0,0 +1,71 @@
+
+## Output files
+
+*Nomadic* produces several files that provide information about the quality of your sequencing run, and the variants that were detected. They can be found in the results directory (`results/<expt_name>`), and are described briefly below:
+
+### `summary.bam_flagstats.csv`
+The `summary.bam_flagstats.csv` contains information about read mapping for each sample.
+
+Each row corresponds to a sample, which can be identified by its barcode (e.g. `barcode01`). The values provided come from [`samtools flagstats`](https://www.htslib.org/doc/samtools-flagstat.html) and are counts of read mapping flags inside of the sample's BAM file. Together they summarise how well your sequencing reads mapped to your reference genome.
+
+| Column | Description |
+| --- | --- |
+| `barcode` | Sample barcode. |
+| `n_total` | Total number of read alignments. |
+| `n_mapped` | Number of mapped read alignments. |
+| `n_unmapped` | Number of unmapped reads. |
+| `n_primary` | Number of reads mapping uniquely. |
+| `n_secondary` | Number of reads mapping to more than one location. |
+| `n_chimera` | Number of reads mapping as chimeras. |
+
+
+### `summary.bedcov.csv`
+The `summary.bedcov.csv` file contains information about sequencing coverage over each amplicon in each sample.
+
+
+Each row contains information about coverage over a specific amplicon in a specific sample. The sample is indicated by its barcode (e.g. `barcode01`) and the amplicon is indicated by its `name` (as well as position and length). The values provided come from [`samtools bedcov`](https://www.htslib.org/doc/samtools-bedcov.html).
+
+| Column | Description |
+| --- | --- |
+| `barcode` | Sample barcode. |
+| `chrom` | Chromosome of amplicon. |
```

Comment:
Not sure, just some thoughts:
is it clear to everyone who will read this what an amplicon is?
What about the word genomic region, when is which word more appropriate? From my understanding a bed file is just a list of genomic regions. For our case those are the amplicons, but could it also be that for someone it will not match?